panda.config.ts

```diff
@@ -59,12 +59,27 @@ const durations = Object.entries(durationsConfig).reduce(durationsReducer, {})
 
 const hideCaret = {
   '0%': {
+    '--faux-caret-opacity': 1,
+    '--faux-caret-line-start-opacity': 1,
+    '--faux-caret-line-end-opacity': 1,
+    '--faux-caret-note-line-end-opacity': 1,
+    '--faux-caret-note-line-start-opacity': 1,
```

Comment:
Can you clarify why the note needs its own line start/end vars rather than using `--faux-caret-note-line-start-opacity`? I suspect that it is to ensure that the faux caret is rendered when shifting from a note to a thought, and a thought to a note, but please confirm.